packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/useLocationDetailView.ts

```diff
@@ -93,11 +96,14 @@ export function useLocationDetailView(
   const { fileDataItems } = locationItems;
   const hasInvalidPrefix = isUndefined(prefix);
 
-  const getConfig = useGetActionInput();
+  const [_, handleDownload] = useProcessTasks(downloadHandler);
 
   const [{ data, isLoading, hasError, message }, handleList] = useDataState(
     listLocationItemsAction,
-    { items: [], nextToken: undefined }
+    {
+      items: [],
+      nextToken: undefined,
+    }
```

Comment:
We probably need to force a prettier version or something along those lines hehe